flutter/example/pubspec.yaml

```diff
@@ -0,0 +1,78 @@
+name: flutter_sample_showcase
+description: A new Flutter project.
```

Comment:
We should probably update the description to match the functionality, something along the lines of: "An example of a Flutter project using Fluent System Icons" or something similar.

Reply:
I agree, updated in 44303dd